### PR TITLE
[ADD] [10.0] barcode_multiline

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

**Review by gjotten — APPROVED** [review]
LGTM functionally, works well customer-side

File: barcodes_multiline/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Barcodes - Multiline",
+    "summary": "It allows barcodes to span multiple lines.",
+    "version": "10.0.1.0.0",
+    "author": "Sunflower IT, "
+              "Odoo Community Association (OCA)",
+    "website": "https://odoo-community.org/",
+    "license": "AGPL-3",
+    "category": "Extra Tools",
+    "depends": [
+        'barcodes',
```
Comment (NL66278) [inline barcodes_multiline/__manifest__.py]: Please use double quotes everywhere.

File: barcodes_multiline/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Sunflower IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline barcodes_multiline/__manifest__.py]: Make license lines https:

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Looks mostly good to me, but few changes requested.

**Comment by thomaspaulb** [comment]
@NL66278 I've attended your comments and added tests.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Comment by thomaspaulb** [comment]
@LoisRForgeFlow This is ready for merge.

**Comment by thomaspaulb** [comment]
@pedrobaeza maybe?

**Pull request merged**